app/code/community/Aligent/TimedBanners/Helper/Data.php

```diff
@@ -1,2 +1,58 @@
 <?php
-class Aligent_TimedBanners_Helper_Data extends Mage_Core_Helper_Abstract { }
\ No newline at end of file
+/**
+ * Class Aligent_TimedBanners_Helper_Data
```

Comment:
@toddhainsworth Would you mind fleshing out each of the phpDoc comments with a short description of the method, any parameters and return types?  The class header should also include a short description of the class itself.